docs/_templates/search.html

```diff
@@ -1,42 +1,79 @@
+    <div class="admonition error">
+      <p class="admonition-title">{% trans %}Error{% endtrans %}</p>
+      <p>
+        {% trans %}Please activate JavaScript to enable the search
+        functionality.{% endtrans %}
+      </p>
+    </div>
+  </noscript>
+  {% block searchtext %}
+  <div>
+    {% trans %}Searching for multiple words only shows matches that contain all
+    words.{% endtrans %}
   </div>
-  <script>
-    // Activate the search field on page load
-    let searchInput = document.querySelector("form.bd-search input");
-    if (searchInput) {
-        searchInput.focus();
-        searchInput.select();
-        console.log("[PST]: Set focus on search field.");
-    }
-  </script>
-{% endblock docs_body %}
-{# Below sections just re-create the behavior of Sphinx default search #}
-{# Page metadata #}
-{%- block htmltitle -%}
-  <title>{{ _("Search") }} - {{ title or docstitle }}</title>
-{%- endblock htmltitle -%}
-{# Manually include the search JS that Sphinx includes #}
-{% block scripts -%}
-  {{ super() }}
-  <script src="{{ pathto('_static/searchtools.js', 1) }}"></script>
-  <script src="{{ pathto('_static/language_data.js', 1) }}"></script>
-  <script src="{{ pathto('searchindex.js', 1) }}"></script>
+  {% endblock %}
+</div>
+
+<script>
+  document.addEventListener("DOMContentLoaded", () => {
+    const queryInput = document.getElementById("search-input").value;
+
+    const script = document.createElement("script");
+    script.src = "https://cdn.nuclia.cloud/nuclia-video-widget.umd.js";
+    script.async = true; // adding this to maybe load nuclia widget faster
+
+    // Initialize the Nuclia widget once it's loaded
+    script.onload = () => {
+      const widget = document.getElementsByTagName("nuclia-search-bar")[0]; // get the widget
+      // Pass the query to the Nuclia Search Widget
+      widget.search(queryInput);
+
+      const script2 = document.createElement("script"); // Call searchtester script after we get search results
+      script2.src = "{{ pathto('_static/searchtester.js', 1) }}";
+
+      // On searchtester.js load call function to add breadcrumbs 
+      script2.onload = () => {
+        addBreadcrumbsToResults();
+      };
+      document.head.appendChild(script2);
+    };
+
+    document.head.appendChild(script);
+  });
+</script>
+
+ <!-- Nuclia Widget -->
+<nuclia-search-bar mode="dark"
+  knowledgebox="62407006-2711-4631-9c03-761d156de289"
+  zone="europe-1"
+  features="answers,navigateToLink,displayMetadata" ></nuclia-search-bar>
+
+<!-- Search results -->
+<nuclia-search-results mode="dark"></nuclia-search-results>
+
+<!-- Optional script -->
+<script>
+  // Activate the search field on page load 
+  let searchInput = document.querySelector("form.bd-search input");
+  if (searchInput) {
+    searchInput.focus();
+    searchInput.select();
+    console.log("[PST]: Set focus on search field.");
+  }
+</script>
+<!-- I Dont think below code is required anymore as it is related to sphinx-->
```

Comment:
I partially agree, but the whitespace changes make it difficult to comment. Please revert the whitespace changes, and `@` me when I can review again. I think that the `<script>` tags for `searchtools.js` and `searchindex.js` may be removed from the template. I am not sure about `language_data.js`.